### PR TITLE
Add sneaker infrastructure

### PR DESCRIPTION
**Pull request by Vektah**
This creates an s3 bucket and a KMS master key to store encrypted application secrets using sneaker.

see https://github.com/codahale/sneaker

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 PREFIX=github.com/99designs/ecs-cli
 VERSION=$(shell git describe --tags --candidates=1 --dirty 2>/dev/null || echo "dev")
 FLAGS=-X main.Version=$(VERSION)
+CFOO=docker run -v `pwd`/templates:/templates vektah/cfoo
```
Comment (joho) [inline Makefile]: what's this for?

**Comment by joho** [comment]
The amazonish bits _look_ ok?

File: Makefile (hunk 1 above)
Comment (Vektah) [inline Makefile]: Its just cfoo in a docker container so I dont need to install ruby and cfoo on my desktop to rebuild the cfn templates.

**Comment by Vektah** [comment]
Probably not the future of config deploy.

**Pull request closed without merging**